test/unit/ngsiv2/examples/contextRequests/updateContext1.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
updateContext1.json -> updateContext.json ?

(Just a suggesttion, current name is also ok; use NTC if you don't plan to change)

Reply:
Fixed in d1bc5ac94a84def33800eb14ecbe95b18a7912bc